doc/en/components/grids/grid/groupby.md

```diff
@@ -69,18 +116,40 @@ public ngOnInit() {
 
 <!-- WebComponents -->
 ```typescript
-connectedCallback() {
-    grid.columns.forEach((column) => {
-        column.groupable = true;
-    });
-}
+    constructor() {
+        var column1 = (this.column1 = document.getElementById("column1") as IgcColumnComponent);
+        column1.groupable = true;
+    }
 ```
 
 During runtime the expressions are gettable and settable from the `groupingExpressions` property. If you need to add or change an existing expression you may also use the `GroupBy` method with either a single or an array of `ISortingExpression`.
 
 ```typescript
 grid.groupBy({ fieldName: 'ProductName', dir: SortingDirection.Desc, ignoreCase: true });
 ```
+<!-- end: WebComponents -->
+
+<!-- Blazor -->
+
```

Comment:
During runtime the expressions are gettable and settable from the `groupingExpressions` property. If you need to add or change an existing expression you may also use the `GroupBy` method with either a single or an array of `ISortingExpression`.
 This is info is missing for blazor here